### PR TITLE
chore: update local snapshots

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

This PR bulk update the snapshots that can be used locally on Mac when running the e2e tests locally.

### CODE REVIEW

File: e2e/busy-screen.e2e.ts-snapshots/Should-hide-spinner-1-Google-Chrome-darwin.png
Comment (dskloetd) [inline e2e/busy-screen.e2e.ts-snapshots/Should-hide-spinner-1-Google-Chrome-darwin.png:1]: I didn't know these exists. Is it really worth having these if we don't test in CI that they are up to date?

**Review by dskloetd — APPROVED** [review]

File: e2e/busy-screen.e2e.ts-snapshots/Should-hide-spinner-1-Google-Chrome-darwin.png
Comment (dskloetd) [reply e2e/busy-screen.e2e.ts-snapshots/Should-hide-spinner-1-Google-Chrome-darwin.png:1]: Sounds good

Comment (peterpeterparker) [reply e2e/busy-screen.e2e.ts-snapshots/Should-hide-spinner-1-Google-Chrome-darwin.png:1]: Will do.

Comment (peterpeterparker) [reply e2e/busy-screen.e2e.ts-snapshots/Should-hide-spinner-1-Google-Chrome-darwin.png:1]: (in another PR)

Comment (peterpeterparker) [reply e2e/busy-screen.e2e.ts-snapshots/Should-hide-spinner-1-Google-Chrome-darwin.png:1]: 👉 #463

**Pull request merged**